docs/software/sciapps/quantumespresso.md

```diff
@@ -1,4 +1,66 @@
 [](){#ref-uenv-quantumespresso}
 # Quantum ESPRESSO
-!!! todo
-    complete docs
+
+Quantum ESPRESSO is an integrated suite of Open-Source computer codes for electronic-structure calculations and materials modeling at the nanoscale. It is based on density-functional theory, plane waves, and pseudopotentials:
+
+PWscf (Plane-Wave Self-Consistent Field)
+FPMD (First Principles Molecular Dynamics)
+CP (Car-Parrinello)
+
+## ALPS (GH200)
+### Setup
+Download the uenv image for QuantumESPRESSO:
+
+
+### List available images
+
+```bash
+uenv image find quantumespresso
+```
+
+### Pull the image of interest
+
+```
+uenv image pull quantumespresso/v7.4:v2
+```
+
+
+QuantumESPRESSO can be compiled from source using the above uenv. The procedure is described in https://eth-cscs.github.io/alps-uenv/uenv-qe/#building-a-custom-version.
```

Comment:
I think the idea is to port over that documentation here. Would you mind moving it? I'd put it after the "how to run" one (as for the CP2K docs).

Reply:
Ok, good point I'll add the how to build